src/test/scala/chiselTests/experimental/ProbeSpec.scala

```diff
@@ -0,0 +1,55 @@
+package chiselTests.experimental
+
+import chisel3._
+import chisel3.experimental.Probe
+import chiselTests.{ChiselFlatSpec, Utils}
+import circt.stage.ChiselStage
+
+class ProbeSpec extends ChiselFlatSpec with Utils {
```

Comment:
Can I see a test of a Bundle `:<>=` to another Bundle, where both contain a field of a probe type?

Reply:
Also do `:=` and `<>`

Reply:
As well as just trying to `:=` two probe types.